pint/models/parameter.py

```diff
@@ -58,44 +61,49 @@ class Parameter(object):
 
     def __init__(self, name=None, value=None, units=None, description=None,
                  uncertainty=None, frozen=True, aliases=None, continuous=True,
+                 parse_value=fortran_float, print_value=str, set_value=lambda x: x,
+                 get_value=lambda x: x, get_num_value=lambda x: x,
                  prior=priors.Prior(priors.UniformRV()),
-                 parse_value=fortran_float, print_value=str,
-                 get_value=lambda x: x, get_num_value=lambda x: x):
+                 parse_uncertainty=None):
+
         self.name = name  # name of the parameter
         self.units = units  # parameter unit in string format,or None
         # parameter num unit, in astropy.units object format.
         # Once it is speicified, num_unit will not be changed.
-
-        self.get_num_value = get_num_value # Method to get num_value from value
-        self.get_value = get_value # Method to update value from num_value
-        self.value = value # The value of parameter, internal storage
-        
-        self.prior = prior 
+        self.set_value = set_value
+        # Method to get num_value from value
+        self.get_num_value = get_num_value
+        self.get_value = get_value  # Method to update value from num_value
+        self.parse_value = parse_value  # method to read a value from string,
+                                        # user can put the speicified format
+                                        # here
+        self.print_value = print_value  # method to convert value to a string.
+        if parse_uncertainty is None:
+            self.parse_uncertainty = fortran_float
+        else:
+            self.parse_uncertainty = parse_uncertainty
```

Comment:
Would it not make more sense to just do self.parse_uncertainty = parse_uncertainty, and set the value of parse_uncertainty to fortran_float by default? Seems shorter to me